source/index.txt

```diff
@@ -33,6 +33,10 @@ database:
   features such as LDAP and Kerberos support, on-disk encryption, 
   and auditing.
 
+- `MongoDB Atlas <https://www.mongodb.com/cloud/atlas?jmp=docs_server>`__
```

Comment:
The ticket specifies that we ought to use `tck` instead of jmp here -- maybe you know something I don't (from the comments, it looks like you might just want to cover that in a separate ticket)? :smile:

Reply:
Yeh, we're going to replace it globally which is a bigger issue and for revertability I'll do it in a separate commit.